main.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/python3
+# main.py
+import sys
+from parser import parse
+from graph import Graph
+
+if len(sys.argv) != 2:
+    raise Exception('Invalid Input, must be given a file which is in the conllu format')
```

Comment:
What about raising `ValueError` instead? Also, it should be "CoNLL-U".

Reply:
Ok good idea